docs/setup/install/README.md

```diff
@@ -1,169 +1,48 @@
-```
-
-#### Access grafana
-
-**Grafana URL**
-
-grafana is available at `BASE_URL/grafana`
-
-**Login credentials**
-
-To log into grafana use username: `admin` and for password run command mentioned below.
-
-```bash
-kubectl -n devtroncd get secret devtron-grafana-cred-secret -o jsonpath='{.data.admin-password}' | base64 -d
-```
-
-#### Configuration
-
-**Configure Secrets**
-
-For `helm` installation this section referes to _**secrets.env**_ section of `values.yaml`. For `kubectl` based installation it refers to `kind: secret` in _**install/devtron-operator-configs.yaml**_.
-
-Following properties should be configured
-
-| Parameter | Description | Default |
-| ---: | :--- | :--- |
-| **POSTGRESQL\_PASSWORD** | Using this parameter the auto-generated password for postgres can be edited as per requirement(Used by devtron to store the app information) | |
-| **WEBHOOK\_TOKEN** | If you want to continue using jenkins for CI then provide this for authentication of requests, should be base64 encoded |  |
-
-**Configure ConfigMaps**
-
-For `helm` installation this section refers to _**configs**_ section of `values.yaml`. For `kubectl` based installation it refers to `kind: ConfigMap` in _**install/devtron-operator-configs.yaml**_.
-
-Following properties should be configured
-
-| Parameter | Description | Default |
-| ---: | :--- | :--- |
-| **BASE\_URL\_SCHEME** | either of http or https \(required\) | http |
-| **BASE\_URL** | url without scheme and trailing slash, this is the domain pointing to the cluster on which devtron platform is being installed. For example if you have directed domain `devtron.example.com` to the cluster and ingress controller is listening on port `32080` then url will be `devtron.example.com:32080` \(required\) | `change-me` |
-| **DEX\_CONFIG** | dex config if you want to integrate login with SSO \(optional\) for more information check [Argocd documentation](https://argoproj.github.io/argo-cd/operator-manual/user-management/) |  |
-| **EXTERNAL\_SECRET\_AMAZON\_REGION** | AWS region for secret manager to pick \(required\) |  |
-| **PROMETHEUS\_URL** | url of prometheus where all cluster data is stored, if this is wrong, you will not be able to see application metrics like cpu, ram, http status code, latency and throughput \(required\) |  |
+* [Devtron](install-devtron.md): The base installation of Devtron includes basic Helm charts. Devtron base install helps to deploy, observe, manage, and debug existing Helm applications in all the clusters.
+* [Devtron with CI/CD](install-devtron-with-cicd.md): This mode includes the base install and the CI/CD module. The CI/CD module installation helps to perform CI/CD, security scanning, GitOps, Access control, debugging, and observability.
 
-#### Storage for Logs and Cache
+## Recommended resources
 
-AWS SPECIFIC
+The minimum requirements for Devtron basic and Devtron with CI/CD installation in production and non-production environments include:
```

Comment:
Remove `basic`, let's limit the phrases `base` , `basic` for discussions per se. In the documents, let's stick to Devtron and Devtron with CICD module.